File: PJDesignBackStageClient/src/app/features/unit/feature-shared/models/create-or-update-unit.ts
import { TemplateType } from "src/app/shared/models/enums";

export class CreateOrUpdateUnitRequest {
    id?: number;
    parent?: number;
    name: string;
    templateType: TemplateType;
    url?: string;
    isAnotherWindow: boolean;
    isEnabled: boolean;
}

File: PJDesignBackStageAPI/App.Model/Unit/CreateOrUpdateUnit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace App.Model
{
    public class CreateOrUpdateUnitRequest
    {
        public int? Id { get; set; }
        public int? Parent { get; set; }
        public string Name { get; set; } = null!;
        public int TemplateType { get; set; }
        public string? Url { get; set; }
        public bool IsAnotherWindow { get; set; }
        public bool IsEnabled { get; set; }
    }
}
